templates/default/slurm.conf.erb

```diff
@@ -88,22 +59,9 @@ SlurmctldLogFile=/var/log/slurmctld.log
 SlurmdDebug=3
```

Comment:
let's update the debug config options to use the proper values (I believe setting the log level as a number comes from very old versions of slurm)